### ISSUE TITLE
Controls not passing arg prop values to storyObj.

### ISSUE THREAD

**Issue by unionen** [JoJk0/storybook-addon-vue-slots#8]
Since the 0.9.24 update story `argTypes` controls are not updating args prop values.

**Comment by amritk** [comment]
Yep confirmed this doesn't work for me either, going to downgrade

**Comment by khylias** [comment]
Additionnal case: Props are passedby String type.

In my case, I've few boolean props and all of them trigger some warning : `Invalid prop: type check failed for prop "isLoading". Expected Boolean, got String with value "false". ` 

Storie.js
```js
argTypes: {
  isLoading: { control: "boolean" },
}
```

Component.vue
```js
props: {
  isLoading: {
    type: Boolean
  }
}
```

**Comment by lasseitorp** [comment]
@khylias I still have the same issue after your patch in 0.9.25 and running Storybook 7.5.3. Downgrading to 0.9.21 still works.

**Comment by filiphazardous** [comment]
I found the problem. If you look at line 8 of https://github.com/JoJk0/storybook-addon-vue-slots/blob/main/src/render.ts - it will always enter the value directly interpreted as a `string`. When it is an `object` - it will be entered as `[object Object]`. Will provide a PR with a suggested fix in a couple of hours.

**Comment by filiphazardous** [comment]
@JoJk0 Another PR to hopefully solve all scenarios of this issue: https://github.com/JoJk0/storybook-addon-vue-slots/pull/11
